src/Tizen.NUI/src/public/Layouting/GridLayout.cs

```diff
@@ -43,369 +97,291 @@ public enum Orientation
             Vertical
         }
 
-        private Orientation _GridOrientation = Orientation.Vertical;
-        const int AUTO_FIT = -1;
-        private int _columns = 1;
-        private int _rows = 1;
-        private int _totalWidth;
-        private int _totalHeight;
-        private int _requestedColumnWidth = 1;
-        private int _numberOfRequestedColumns;
-        private int _requestedRowHeight = 1;
-        private int _numberOfRequestedRows;
-        internal GridLocations _locations;
-
+        /// <summary>
+        /// [Draft] Enumeration for the alignment of the grid layout items
+        /// </summary>
+        public enum Alignment
+        {
+            /// <summary>
+            /// At the start of the container.
+            /// </summary>
+            Start,
+            /// <summary>
+            /// At the end of the container.
+            /// </summary>
+            End,
+            /// <summary>
+            /// At the center of the container
+            /// </summary>
+            Center,
+            /// <summary>
+            /// Fill the container
+            /// </summary>
+            Fill,
+        }
 
         /// <summary>
-        /// [Draft] Get/Set the orientation in the layout
+        /// Get the column index.
         /// </summary>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Orientation GridOrientation
+        public static int GetColumn(BindableObject bindable)
         {
-            get
-            {
-                return _GridOrientation;
-            }
-            set
-            {
-                _GridOrientation = value;
-                RequestLayout();
-            }
+            return (int)bindable.GetValue(ColumnProperty);
         }
 
         /// <summary>
-        /// [draft] GridLayout Constructor/>
+        /// Get the column span.
         /// </summary>
-        /// <returns> New Grid object.</returns>
-        /// <since_tizen> 6 </since_tizen>
-        public GridLayout()
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static int GetColumnSpan(BindableObject bindable)
```

Comment:
If only View can be param of this function, I think it is easy to understand using View as parameter instead of BindableObject.

Reply:
Nice catch. i will update.